core/client.go

```diff
@@ -0,0 +1,51 @@
+package core
+
+import (
+	"fmt"
+
+	"github.com/celestiaorg/celestia-core/config"
+	corenode "github.com/celestiaorg/celestia-core/node"
+	"github.com/celestiaorg/celestia-core/rpc/client"
+	"github.com/celestiaorg/celestia-core/rpc/client/http"
+	"github.com/celestiaorg/celestia-core/rpc/client/local"
+)
+
+// Client is an alias to Core Client.
```

Comment:
```suggestion
// Client is an alias to Core Client. A core Client can be initialised as an embedded
// process or created by dialing a given remote endpoint. The default behaviour is 
// embedding.
```

Reply:
There is no default behavior for an alias definition, so the last sentence is incorrect and out of context for `Client` godoc.